scheduler/server/cluster_state.go

```diff
@@ -214,11 +217,18 @@ func (c *clusterState) updateCluster() {
 	defer c.stats.Latency(stats.SchedUpdateClusterLatency_ms).Time().Stop()
 	select {
 	case updates, ok := <-c.updateCh:
+		elapsed := time.Since(c.priorUpdateTime)
+		log.Infof("got %d updates off updateCh.  Time since last update: %s", len(updates), elapsed)
+		if c.stats != nil {
+			c.stats.Gauge(stats.ClusterStateTimeSinceLastUpdate_ms).Update(elapsed.Milliseconds())
+		}
+		c.priorUpdateTime = time.Now()
 		if !ok {
 			c.updateCh = nil
 		}
 		c.update(updates)
 	default:
+		log.Info("no updates were on the channel")
```

Comment:
nit: Should we use log.Debug() here to avoid adding extra log lines(if this is not important to know in every step)?

Reply:
Generally yes, this would be more of a debug type log, and it's a pretty lousy thing to be logging to be honest. However for a one-off deploy this will help us debug what's going on with delays here; we won't ever merge this code into master. We also only log Debug level on staging, and we need to see this behavior on prod, so has to be Info or higher here.

Reply:
Oh, I see. That makes sense.